en/administration/configuration/config-file-reference.md

```diff
@@ -266,29 +266,29 @@ system and is useful when managing Rundeck server clusters.
 
 ### Project Configuration Storage settings
 
-The [Project Setup - Project Definitions](project-setup.html#project-definitions) mechanism is configured within this file, see:
+The [Project Setup - Project Definitions][page:administration/projects/project-create.md#project-definitions] mechanism is configured within this file, see:
 
 * [Project Storage][]
 
-[Project Storage]: storage-facility.html#project-storage
+[Project Storage]: page:administration/configuration/storage-facility.md#project-storage
```

Comment:
Links like this are not going to work... it is expecting a url here.

~~~
[link text][]

[link text]: page:blahblah
~~~

This needs to be changed to just

~~~
[link text][page:blahblah]

~~~

This is because the edam script is adding `[page:blahblah]: the/actual/path` as a markdown link reference in the document footer already, and i don't think the markdown link reference syntax allows one reference to use another reference.

Reply:
fyi: i also see this repeated in 40 places.  searching for ` page:` (with the space)

Reply:
Ahh, that makes sense.  I didn't realize that's how edam worked.  That means I'll have to do two passes, but that shouldn't be hard.

Actually, rather than make the current script do two passes, I'll write a script to do the second pass: look for the `[link text]: page:...` lines and remove them, then replace the `[link text][]` with the correct embedded link.

This does mean that going forward, folks can't write the link reference footer syntax for internal links, but must always inline them.  Not sure if that'll be surprising to anyone, since most people don't even know about the link reference footer syntax.  :)

Reply:
edam also handles the `[[page:blah]]` internally, replacing it with `(page title)[page:blahblah]`, perhaps it could be updated too to look for the paragraph style references

Reply:
I believe all of the paragraph link references have been updated to appar as `[link text][page:blahblah]`